src/antelope/types/EvmRexDeposit.ts

```diff
@@ -0,0 +1,6 @@
+import { ethers } from 'ethers';
+
+export interface EvmRexDeposit {
+    amount: ethers.BigNumber;
+    until: ethers.BigNumber;
```

Comment:
can you leave a comment about what `until` is?

Reply:
added